mllib/src/main/scala/org/apache/spark/mllib/fpm/AssociationRules.scala

```diff
@@ -142,5 +142,13 @@ object AssociationRules {
     def javaConsequent: java.util.List[Item] = {
       consequent.toList.asJava
     }
+
+    /**
+     * @return [antecedent] => [consequent]: [confidence]
```

Comment:
Actually, I think `Returns` is better than `@return`. If we use `@return`, the message doesn't show up by default in the API doc, e.g.,

![screen shot 2015-09-24 at 8 38 56 am](https://cloud.githubusercontent.com/assets/829644/10078054/e65b67f2-6297-11e5-8cdc-e691b382f740.png)

User needs to open it to see the doc:

![screen shot 2015-09-24 at 8 39 05 am](https://cloud.githubusercontent.com/assets/829644/10078062/f0e54026-6297-11e5-80a3-02ac94c778b5.png)

If we use `Returns`, it shows up by default:

![screen shot 2015-09-24 at 8 39 35 am](https://cloud.githubusercontent.com/assets/829644/10078069/fae54be8-6297-11e5-99c1-48108442e0ad.png)

@srowen